package.json

```diff
@@ -10,14 +10,18 @@
     "lint": "yarn lint:js && yarn lint:css && yarn lint:md",
     "lint:css": "stylelint 'src/**/*.{js,ts,tsx}'",
     "lint:js": "eslint ./ --ext js,ts,tsx --max-warnings 0",
-    "lint:md": "markdownlint ./*.md src/**/*.md",
+    "lint:md": "markdownlint ./*.md src/**/*.md content/**/*.{md,mdx}",
```

Comment:
Should `mdx` be added to the list under the `format:all` prettier script command? Also prefer combined `{content,src}/**/*.{md,mdx}` here.

Reply:
Updated.